packages/callout/src/Callout.tsx

```diff
@@ -0,0 +1,229 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import BulbIcon from '@leafygreen-ui/icon/dist/Bulb';
+import ImportantWithCircleIcon from '@leafygreen-ui/icon/dist/ImportantWithCircle';
+import InfoWithCircleIcon from '@leafygreen-ui/icon/dist/InfoWithCircle';
+import WarningIcon from '@leafygreen-ui/icon/dist/Warning';
+import { LGGlyph } from '@leafygreen-ui/icon/dist/types';
+
+export const Variant = {
+  Note: 'note',
+  Tip: 'tip',
+  Important: 'important',
+  Warning: 'warning',
+} as const;
+
+export type Variant = typeof Variant[keyof typeof Variant];
+
+const baseStyle = css`
+  border-radius: 6px;
+  box-shadow: 0px 2px 5px 0 rgba(33, 49, 60, 0.1);
```

Comment:
might be nice to have a help like this so we can re-use the palette colors
https://github.com/Chris-Baker/Emotion-RGBA

Reply:
Already have one! We use [Polished](https://polished.js.org/), and it's included via lib. It has a helper for this.

https://github.com/mongodb/leafygreen-ui/blob/master/packages/lib/package.json#L24